usagov/vote-gov-drupal
# Feature/vote 1109/korean state character logic

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1109

## Description (optional)
Rework the Korean state character logic to use translation files

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. lando retune

### Testing steps
1. Visit http://vote-gov.lndo.site/ko/register/as and verify the state char is not displayed (Korean - territory)
2. Visit http://vote-gov.lndo.site/ko/register/co and verify the state char is displayed (Korean - state)
3. Visit http://vote-gov.lndo.site/register/co and verify that the "state" string does not display (English - state)
